src/Microsoft.AspNetCore.SignalR/HubConnection.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.IO.Pipelines;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Sockets;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubConnection : IClientProxy
+    {
+        private object _lock = new object();
+        private Task _taskQueue = TaskCache.CompletedTask;
```

Comment:
Does this mean that when not using HubConnection you have to do this on your own? Can't the queue be not coupled to the HubConnection and be reusable?